src/events/guild/roles/roleUpdate.ts

```diff
@@ -15,11 +18,15 @@ export default class extends Event {
 	}
 
 	public async run(oldRole: Role, newRole: Role): Promise<void> {
-		const difference = this.client.bulbutils.diff(oldRole, newRole).filter(k => k !== "rawPosition")
-		if(difference.length === 0) return;
+		const config: GuildConfiguration = await databaseManager.getConfig(newRole.guild.id);
+		if (newRole.rawPosition > newRole.guild.me.roles.highest.rawPosition && newRole.id === config.muteRole) await databaseManager.setMuteRole(newRole.guild.id, null);
```

Comment:
Use a nested `if` for this, the first two conditions are the same so they only need to be checked once.